include/nuttx/fs/fs.h

```diff
@@ -352,6 +352,8 @@ struct inode
   int16_t           i_crefs;    /* References to inode */
   uint16_t          i_flags;    /* Flags for inode */
   union inode_ops_u u;          /* Inode operations */
+  uid_t             i_uid;      /* The ID of the inode owner */
+  gid_t             i_gid;      /* The ID of the inode owner group */
 #ifdef CONFIG_FILE_MODE
```

Comment:
Is the initial value zero?  That is the root user and group.  Shouldn't the initial value be the user and group ID of the task creating the inode?

Reply:
You are right. These two variables are added to the header file. When creating inode, these two values are to be set as the user and group ID of the task. Therefore, more work needs to be done in terms of permissions